devex/vscode-extension/src/common/constants.ts

```diff
@@ -10,6 +10,11 @@ export class Constants {
     //
     public static ExtensionId = "ms-iot.msiot-vscode-openenclave";
 
+    //
+    // SDK version support
+    //
+    public static SdkVersion = "v0.11.0";
```

Comment:
why introduce another version?  what does DevKitVersion mean with SdkVersion?

Reply:
Oh, interesting. Would it then make sense to merely bump `DevKitVersion` to `3.0.0.0`? It seems to me that I reinvented a solution you had already implemented for the same problem.

Reply:
I've removed the code I added and just bumped `DevKitVersion` to `3.0.0`; it does the trick. Thank you for catching this!